uw-frame-components/portal/notifications/partials/notifications-list-item.html

```diff
@@ -0,0 +1,23 @@
+<!-- NOTIFICATION CONTENT -->
+<a class="md-list-item-text"
+   ng-href="{{ notification.actionURL }}"
+   target="_blank"
+   aria-label="{{ notification.actionAlt }}"
+   layout="row"
+   layout-align="start center"
+   layout-xs="column"
+   layout-align-xs="center start">
+  <span><i class="fa fa-exclamation-triangle" ng-if="notification.priority"></i> {{ notification.title }}</span>
```

Comment:
Maybe if we use ALL CAPS for priority notifications, screen readers will yell the content. ;)